laravel-app/database/migrations/0001_01_01_000000_create_users_table.php

```diff
@@ -16,6 +16,7 @@ public function up(): void
             $table->string('name');
             $table->string('email')->unique();
             $table->timestamp('email_verified_at')->nullable();
+            $table->unsignedInteger('company_id');
             $table->string('password');
```

Comment:
- 一応統一性を設けるためにcompany_idもunsignedBigIntegerに
`流石に42億データはは超えないけど。。。`

```suggestion
            $table->timestamp('email_verified_at')->nullable();
            $table->unsignedBigInteger('company_id');
            $table->string('password');
```